cli/src/cmd/install.rs

```diff
@@ -79,12 +79,19 @@ pub(crate) fn install(
 
 /// installs the dependency as an ordinary folder instead of a submodule
 fn install_as_folder(dep: &Dependency, libs: &Path) -> eyre::Result<()> {
-    Command::new("git")
+    let result = Command::new("git")
         .args(&["clone", &dep.url, &dep.name])
         .current_dir(&libs)
         .stdout(Stdio::piped())
         .spawn()?
-        .wait()?;
+        .wait();
```

Comment:
This can be replaced with a `.status()?` instead (same goes for below)

Reply:
oh nice, updating

Reply:
Ah, I should have been clearer: both `spawn()?` and `wait()` can be merged into one `.status()?` which returns a `Result<ExitStatus>` (where the `Err` variant is if the process could not be spawned)